src/library/librarycontrol.cpp

```diff
@@ -557,28 +580,41 @@ void LibraryControl::slotToggleSelectedSidebarItem(double v) {
 }
 
 void LibraryControl::slotGoToItem(double v) {
-    if (!m_pLibraryWidget) {
+    VERIFY_OR_DEBUG_ASSERT(m_pSidebarWidget) {
         return;
     }
-    // Load current track if a LibraryView object has focus
-    const auto activeView = m_pLibraryWidget->getActiveView();
-    if (activeView && activeView->hasFocus()) {
-        return slotLoadSelectedIntoFirstStopped(v);
+    VERIFY_OR_DEBUG_ASSERT(m_pLibraryWidget) {
+        return;
     }
 
-    // Focus the library if this is a leaf node in the tree
-    if (m_pSidebarWidget && m_pSidebarWidget->hasFocus()) {
-        if (v > 0 && m_pSidebarWidget->isLeafNodeSelected()) {
-            setLibraryFocus();
-        } else {
-            // Otherwise toggle the sidebar item expanded state
-            slotToggleSelectedSidebarItem(v);
+    if (v > 0) {
+
```

Comment:
Instead of indenting the rest of the function, you could change this to
```
if (v <= 0) {
    return;
}
```

Reply:
Besides that, this PR looks good to me. I tested by modifying my controller script.

Reply:
(this is called using a "guard clause" and it's a really helpful technique: https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)